python-numpy-basics/Classes_and_Objects.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>Add proper python syntax highlighting in second point</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It is visible properly in jupyter notebook. Seems to have some problem here</p>